clj-refactor.el

```diff
@@ -1511,7 +1511,7 @@ front of function literals and sets."
   (interactive)
   (insert "/")
   (when (and cljr-magic-requires
-             (looking-back cljr--magic-requires-re 6))
+             (looking-back (cljr--magic-requires-re) 6))
```

Comment:
This hardcoded 6 seems a bit weird.  Shoudn't that be something like `point-at-bol` instead of the buffer position 6?